cmd/admin-handlers.go

```diff
@@ -2527,6 +2531,135 @@ func fetchKMSStatus() madmin.KMS {
 	return kmsStat
 }
 
+// fetchKMSStatusV2 fetches KMS-related status information for all instances
+func fetchKMSStatusV2() []madmin.KMS {
```

Comment:
We should probably pass a `context.Context` and not use the `context.Background` to allow clients to cancel the request

Reply:
Ack. Will change and update.